_posts/2021-10-04-SE2022-RSE-Track-Call-for-contributions-de.md

```diff
@@ -0,0 +1,65 @@
+
+Das Organisationskomitee freut sich auf Einreichungen für Workshops, Vorträge und Poster, sowie für Splinter Meetings und informelle “Birds-of-a-Feather (BoF)” Diskussionstreffen. Ziel der Konferenz ist die Abbildung der facettenreichen Gemeinschaft der Research Software Engineers durch Beteiligung aller Erfahrungsstufen über die verschiedenen Fachrichtungen, Standorte, Geschlechter und Herkünfte hinweg.
+
+## Timeline
+*	RSE Track: **Einreichunsgfrist  12.11.2021**
+*	RSE Track: Benachrichtigung 3.12.2021
+*	RSE Track: Finale version 15.12.2021 
+*	**Mo-Di. 21. - 22.02.2022 - rSE22 Workshops**
+*	**Mi-Fr., 23. - 25.02.2022 - rSE22 Konferenz**
+
+## Konferenz-Sessions, Themen und Einreichungstypen
+rSE22 ist ein Track der [SE 2022](https://www.se-2022.de) Konferenz der GI mit Fokus auf Forschungssoftware. Die Sessions  innerhalb des Tracks werden anhand der Einreichungen vom Programmkomitee zusammengestellt. Wenn du eine Arbeit zu einem Thema hast, das für die Forschungssoftwaregemeinschaft von allgemeinem Interesse sein könnte, freuen wir uns auf deine Einreichung!
+
+## Wer kann einreichen?
+Wir begrüßen Einreichungen in Deutsch oder Englisch von all jenen, die einen interessanten Beitrag zu Forschungssoftware haben. Dazu gehören unter anderem:
+*	Forschende aller Karrierestufen, die Software zu Forschungszwecken entwickeln
+*	Softwareentwickler*innen, die in Forschungskontexten arbeiten, unabhängig von ihrer Stellenbezeichnung und Disziplin
+*	Förderer und Entscheider*innen, die mit Fördermitteln und anderen Werkzeugen den Weg für nachhaltige Forschungssoftware ebnen
+*	Personen, die an der Verbesserung unseres Verständnisses interessiert sind, wie Forschungssoftware bestmöglich entwickelt und gewartet werden kann, z.B. inm Bezug auf Offenheit, Reproduzierbarkeit, Nachhaltigkeit oder Skalierbarkeit und Performanz
+*	Personen aus Organisationen, die Tools, Plattformen oder Services zur Unterstützung von Forschungssoftware bereit stellen, z.B. Anbieter von IT-Infrastrukturen, Daten- und Rechenzentren
+
+Unser Ziel ist ein ausgewogenes Programm, das die Vielfalt der Perspektiven und die Diversität der RSE-Gemeinschaft und ihrer Aktivitäten abbildet. Bei ähnlichen Konferenzen außerhalb Deutschlands wurde festgestellt, dass die Geistes- und Sozialwissenschaften dort unterrepräsentiert waren, so dass wir RSE-Aktive aus diesen Feldern ausdrücklich zur Einreichung auffordern möchten.
+
+Wir suchen nicht nur nach erfahrenen, bekannten oder promovierten Vortragenden. Daher möchten wir ausdrücklich auch alle Personen aus unterrepräsentierten Gruppen und Early Career Researchers zur Einreichung auffordern!
+
+Einreichungen werden im “Single-Blind”-Verfahren begutachtet, das Software-bezogenen Einreichungen entgegen kommt. Das heißt: Gutachter*innen werden Namen und Herkunftsinstitutionen von Autor*innen einsehen können.
```

Comment:
```suggestion
Einreichungen werden im “Single-Blind”-Verfahren begutachtet, das Software-bezogenen Einreichungen entgegen kommt. Das heißt: Gutachter*innen werden Namen und Herkunftsinstitutionen von Autor*innen einsehen können.
```

Preview here in the PR renders `*` as italics (as there are two in this line).

Reply:
done